type-generator/.gitignore

```diff
@@ -0,0 +1,107 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
```

Comment:
Why does this include Python related ignores?

Reply:
You are actually right. I started this as a python project. I am going to change this 👍